managing_providers/_topics/automation_management_providers.md

```diff
@@ -365,3 +365,174 @@ To use the button to run an Ansible Tower job on a virtual machine:
 If you selected a service dialog to run when creating the button, {{ site.data.product.title_short }} will then prompt you to enter variables to complete the task. After entering your desired parameters, {{ site.data.product.title_short }} takes you to the **Requests** page.
 
 The service item’s details can be viewed in menu: **Services > My Services** in {{ site.data.product.title_short }}.
+
+## OpenTofu
+
+OpenTofu is an open source infrastructure as code tool, which can be used to build, change, and version the infrastructure. You can use OpenTofu to define infrastructure resources in human-readable configuration files that you can use to version, reuse, and share.
+
+OpenTofu is built into {{ site.data.product.title_short }} so you do not need to install any additional components. 
+
+If you want to use the Embedded Terraform feature in {{ site.data.product.title_short }} that is deployed as a virtual machine appliance, then you need to manually import the Terraform image on an appliance:
+
+## Importing OpenTofu image on an appliance
+
+**Note**: Follow this section if you have {{ site.data.product.title_short }} that is deployed as a virtual machine appliance. These steps are not applicable to {{ site.data.product.title_short }} that is deployed as a containerized deployment (podified).
+
+Use the following command to import the OpenTofu image on your appliance server.
+
+``` bash
+runuser --login manageiq --command 'podman --root=/var/www/miq/vmdb/data/containers/storage image load --input /tmp/<OpenTofu_image>'
+```
+
+Where `OpenTofu_image` is the name of your OpenTofu image.
+
+``` bash
+runuser --login manageiq --command 'podman --root=/var/www/miq/vmdb/data/containers/storage image load --input /tmp/infra-mgmt-opentofu-runner_PR-110-204033d-20240521-x86_64.tar'
+```
```

Comment:
I don't think we need this as that image isn't something which would be publicly available (it was from a PR not a release)

Reply:
```suggestion
```